### PR TITLE
Use VCR api v2

### PR DESCRIPTION
**Pull request by reinkrul**

### CODE REVIEW

**Comment by woutslakhorst** [comment]
we also have to fix code example paths afterwards: https://github.com/nuts-foundation/nuts-workshops/search?q=nuts-demo-ehr

File: nuts/client/vcr.go [hunk 1]
```diff
@@ -51,86 +48,70 @@ func (c HTTPClient) CreateVC(ctx context.Context, typeName, issuer string, crede
...
 	_, err = testAndReadResponse(http.StatusOK, response)
 	return err
 }
 
-func (c HTTPClient) ResolveVerifiableCredential(ctx context.Context, credentialID string, untrusted bool) (*vcr.VerifiableCredential, error) {
-	response, err := c.vcr().Resolve(ctx, credentialID, &vcr.ResolveParams{})
+func (c HTTPClient) ResolveCredential(ctx context.Context, credentialType ssi.URI, credentialID string, untrusted bool) (*vc.VerifiableCredential, error) {
+	query := GetNutsCredentialTemplate(credentialType)
+	query.ID, _ = ssi.ParseURI(credentialID)
+	result, err := c.search(ctx, query, untrusted)
```
Comment (woutslakhorst) [inline nuts/client/vcr.go:83]: We have a resolve API now to do this

**Review by woutslakhorst — CHANGES_REQUESTED** [review]
just 1 thing and check comment on workshop code samples

**Comment by reinkrul** [comment]
> we also have to fix code example paths afterwards: https://github.com/nuts-foundation/nuts-workshops/search?q=nuts-demo-ehr

Will add it to the v1 epic TODO list

**Review by woutslakhorst — APPROVED** [review]

**Pull request merged**